strings/tcg/set_one.json

```diff
@@ -464,70 +464,70 @@
 			"id": "ert_command",
 			"name": "NT P.A.V. Suit (Command)",
 			"desc": "Issued to members of Emergency Response Teams, the P.A.V. Suit gives superior protection from any threat the galaxy can throw at it. This particular model is outfitted with a sidearm holster and a sleek blue finish.",
-			"rules": "While equipped, give the equipped unit {$Squad Tactics} and {$First Strike}.",
+			"rules": "While equipped, give the equipped unit {$First Strike}.",
 			"icon_state": "ert_command",
-			"power": "2",
-			"resolve": "2",
+			"power": "0",
+			"resolve": "0",
 			"faction": "Command",
 			"summoncost": "2",
 			"cardtype": "Equipment",
-			"cardsubtype": "Armour",
+			"cardsubtype": "Command ERT",
```

Comment:
Commenting here but it's present in a few different cards, from the looks of it you phased out the Armour card subtype from several cards and replaced it with an appropriate set of subtypes elsewhere, but not on every card. What's your intent?

Reply:
What cards didn't get armor replaced? I do want to phase it out entirely.
The idea is that subtype becomes like MtG's tribes or YGO's archetypes, and decks be mostly tribal. For the long term I want to add a rule so that when building your deck you choose 3 subtypes of cards; all your cards must have at least one of these subtypes. (If a card has 2 or more subtypes then you only need one of them to include that card on your deck). 

This lets us dodge a lot of problems. It helps fight off power creep as now every card isn't in direct competition with eachother, being evaluated in the context of its wider subtype than compared to every other card in the game. It also helps slow down insane wombo combos from appearing, as you can't have 4 card combos from different subtypes, and dedicating your entire deck to a 3 card combo means your subtypes might not have that much synergy with one another.

So eventually I want there to be a ERT strategy, a Bar strategy, a Plant strategy, etc.